router.go

```diff
@@ -380,7 +384,13 @@ func (r *Router) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 		// Handle OPTIONS requests
 		if r.HandleOPTIONS {
 			if allow := r.allowed(path, req.Method); len(allow) > 0 {
+				if r.OPTIONSPostHandler != nil {
+					r.OPTIONSPostHandler.ServeHTTP(w, req)
```

Comment:
Is this still here on purpose?
Why is the _Post_ handler called twice and once _before_?